src/applicationSummary.js

```diff
@@ -105,28 +100,30 @@ export const ApplicationSummary = (props) => {
             </div>
             <div className="text-content">
                 {state.fetchedApplication &&
-                <>
-                    <h1>Application Summary</h1>
```

Comment:
Don't we need the header anymore and have textual changes been approved by Karyn?